games/minetest/Portfile

```diff
@@ -85,20 +103,22 @@ patchfiles-append       003-patch-ignore-psn-option-mac-bundle.diff
 
 configure.args-append   -DCMAKE_BUILD_TYPE=Release \
                         -DCMAKE_INSTALL_PREFIX:PATH=${applications_dir} \
-                        -DBUILD_CLIENT=1 \
-                        -DENABLE_SOUND=1 \
-                        -DENABLE_REDIS=1 \
-                        -DENABLE_POSTGRESQL:BOOL=OFF \
-                        -DENABLE_LEVELDB=1 \
-                        -DENABLE_FREETYPE=1 \
-                        -DENABLE_CURL=1 \
-                        -DENABLE_GETTEXT=1 \
-                        -DENABLE_SYSTEM_JSONCPP=1 \
-                        -DENABLE_UPDATE_CHECKER=0 \
+                        -DBUILD_UNITTESTS=OFF \
+                        -DBUILD_CLIENT=ON \
+                        -DBUILD_SERVER=ON \
+                        -DENABLE_SOUND=ON \
+                        -DENABLE_REDIS=OFF \
```

Comment:
why switch off this option compared to what is in the current Portfile?

Reply:
Rationale:

IMO the REDIS is both somewhat exotic and presumably was not used in the wild anyway.
1. The MT install is using SQLite3 as database default. So redis was built but not used, I presume. Furthermore see at: <https://github.com/minetest/minetest#dependencies>
2. The normal MT player would either use the default (sqlite) or should use LEVELDB for mulit-player as recommend at: <https://wiki.minetest.net/Database_backends>
3. Both SQLITE and LEVELDB are seemless on macOS as macOS provides a legacy sqlite version.
4. Using REDIS would need further configurations by the user i.e. the MT player. This would be too much of a challenge for most MT players, I presume.
5. An expert might like to use REDIS but this is already in my backlog and will be a new variant `minetest --redis` like `mintetest --psql` and `minetest --devel` or `minetest --server` in the future.
6. I could add a variant `minetest --redis` if you insist. But I would prefer an approval of rev 1 first and would like to postpone this effort for the rev 2 if possible.

Agree?